spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  # before { request.env["omniauth.auth"] = OmniAuth.config.mock_auth[:twitter] }
+  let(:user) { create(:developer_user) }
+  let(:creator) { create(:twitter_creator) }
+
+  describe "GET #show" do
+    it "renders the show view" do
+      get :show, {id: user.id}, user_id: user.id
+      expect(subject).to render_template :show
+    end
+    it "assigns current user's creators as @creators" do
+      user.creators << creator
+      creators = user.creators
+      get :show, {id: user.id}, user_id: user.id
+      expect(assigns(:creators)).to eq creators
+    end
+  end
+
+  describe "GET #feed" do
+    context "user is logged in" do
+      let (:params) do
+        {p_id: "154915030",
```

Comment:
Is there any reason why you keep re-creating this `let (:params)` instead of placing them at a higher scope and just defining it once?